### ISSUE TITLE
question2answer is not working on plesk panel.

### ISSUE THREAD

**Issue by Abdur-Raziq-Khan** [q2a/question2answer#820]
I uploaded **question2answer** library to **plesk** panel and kept in the sub-directory of my website. Further I can route to the main page of **question2answer** which is fine, but when I click for "Ask Question" or on other route items which gives me error _"HTTP Error 404.0 - Not Found"_ which means that the file I am looking for does not exist or their is some other problem. The issue may be on server side as it works fine localhost.

**Comment by phy25** [comment]
Is that a Apache or NGINX? You need to set up URL rewrite.

**Comment by Abdur-Raziq-Khan** [comment]
For localhost I am using **xampp**. I hosted my website in [Plesk](https://www.plesk.com/). Also for database I am using **phpmyadmin**, which is connected with **question2answer** and is fine.

**Comment by svivian** [comment]
Plesk is not a server as far as I know, the server will be something like Apache or nginx. If you're using Apache then make sure you rename the `.htaccess-example` file to `.htaccess`. If you're not sure ask the hosting company.

The GitHub repo is not for general install questions, if you have further problems then please ask on the [Q2A site](https://www.question2answer.org/qa/).

**Comment by Abdur-Raziq-Khan** [comment]
> Is that a Apache or NGINX? You need to set up URL rewrite.

I'm using Apache.

**Comment by phy25** [comment]
> If you're using Apache then make sure you rename the .htaccess-example file to .htaccess.

**Comment by Abdur-Raziq-Khan** [comment]
> > If you're using Apache then make sure you rename the .htaccess-example file to .htaccess.

Yes, I did renamed it as well as according to some tutorials, but this is not the case.

**Comment by Abdur-Raziq-Khan** [comment]
This is the [link](http://studypooling.com/question-answer-form/) to my website. You can check it.

**Comment by svivian** [comment]
That’s not Apache. The error page says it’s IIS (Windows) server. Q2A is not officially supported on IIS but I think others have got it working fine in the past. You can try searching for an appropriate web.config file.

In the meantime your site works fine using URLs like `http://studypooling.com/question-answer-form/index.php/questions` so you can try visiting the admin page at http://studypooling.com/question-answer-form/index.php/admin and setting the URL structure option.

**Comment by Abdur-Raziq-Khan** [comment]
> That’s not Apache. The error page says it’s IIS (Windows) server. Q2A is not officially supported on IIS but I think others have got it working fine in the past. You can try searching for an appropriate web.config file.
> 
> In the meantime your site works fine using URLs like `http://studypooling.com/question-answer-form/index.php/questions` so you can try visiting the admin page at http://studypooling.com/question-answer-form/index.php/admin and setting the URL structure option.

So, what should I do then? There is no proper documentation/ tutorials for websites published on plesk.

**Comment by Abdur-Raziq-Khan** [comment]
There is no other such library which I can use. I think **question2answer** is the only solution, but right I am facing such issue with it.
